website_snippet_marginless_gallery/tests/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2015-today Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
I got inspired by https://github.com/facebook/react/commit/bef45b0b1a98ea9b472ba664d955a039cf2f8068 and the assumption that Facebook has good lawyers.